src/cue_notes_create.lua

```diff
@@ -71,52 +75,32 @@ function region_contains_notes(region, layer_number)
 end
 
 function new_cue_name(source_staff)
-    local dialog = finale.FCCustomWindow()
-    local str = finale.FCString()
-    str.LuaString = plugindef()
-    dialog:SetTitle(str)
-
-    str.LuaString = "New cue name:"
-    dialog:CreateStatic(0, 20):SetText(str)
+    local dialog = mixin.FCXCustomLuaWindow():SetTitle(plugindef())
+    dialog:CreateStatic(0, 20):SetText("New cue name:"):SetWidth(100)
 
-    local the_name = dialog:CreateEdit(0, 40)
-    the_name:SetWidth(200)
-    -- copy default name from the source Staff Name
-    local staff = finale.FCStaff()
+    local staff = finale.FCStaff() -- copy the source Staff Name
     staff:Load(source_staff)
-    the_name:SetText(staff:CreateDisplayFullNameString())
-
+    local the_name = dialog:CreateEdit(0, 40):SetWidth(200):SetText(staff:CreateDisplayFullNameString())
     dialog:CreateOkButton()
     dialog:CreateCancelButton()
     local ok = (dialog:ExecuteModal(nil) == finale.EXECMODAL_OK)
+    local str = finale.FCString()
```

Comment:
I think lines 87-89 could be shortened since `FCMControl.GetText` returns a string:
```lua
return ok, the_name:GetText()
```